server/src/main/java/se/fortnox/reactivewizard/server/RwServer.java

```diff
@@ -44,10 +44,13 @@ public class RwServer extends Thread {
         "application/json"
     ));
 
+    private static final int LOGGING_SHUTDOWN_TIMEOUT_SEC = 3;
```

Comment:
Nice changes overall, I think it is a good start! Perhaps the timeout config should be configurable in the ServerConfig?

Reply:
This is a local technical timeout. It does not bear any semantics that would be meaningful for any other stages of the shutdown process. It just guards of extremely unlikely conditions when the logging shutdown does not respond. I thought this parameter would obstruct the ServerConfig with something that is not interesting in the global context. Maybe I'm mistaken. We can add it to the config if you think it's necessary.

Reply:
I want to avoid changing this in the open source RW later on, but I do agree that 3 seconds probably covers all cases so there should be no reason to change this in the future. I somewhat agree that ServerConfig is probably wrong to use here, perhaps just a system property would otherwise work. Nevertheless, no changes needed :)

Reply:
The timeout has been removed. The context is outdated.